picasso/src/main/java/com/squareup/picasso/Picasso.java

```diff
@@ -166,16 +162,66 @@
 
   boolean shutdown;
 
+  /**
+   * This instance is automatically initialized with defaults that are suitable to most
+   * implementations.
+   * <ul>
+   * <li>LRU memory cache of 15% the available application RAM</li>
+   * <li>Disk cache of 2% storage space up to 50MB but no less than 5MB. (Note: this is only
+   * available on API 14+ <em>or</em> if you are using a standalone library that provides a disk
+   * cache on all API levels like OkHttp)</li>
+   * <li>Three download threads for disk and network access.</li>
+   * </ul>
+   * <p>
+   * If these settings do not meet the requirements of your application, you can construct your own
+   * with full control over the configuration by using {@link Picasso.Builder} to create a
+   * customized {@link Picasso} instance.
+   */
+  public Picasso(Context context) {
```

Comment:
Why the extra constructor? You can always do `new Picasso.Builder(context).build()`.
I worry about people who will fire and forget. `new Picasso(context).load().into()`. I'd hope the awkwardness of the builder causes pause; you only have to do it once in the common case!

Reply:
Took a page out of OkHttp and how OkHttpClient is modeled.  Fair point though; let's see what the others think.

Reply:
I would prefer having just the builder because it keeps defaults in one location.

Reply:
removed the proposed added constructor.